corehq/apps/domain/views/settings.py

```diff
@@ -588,12 +611,80 @@ def _create_alert(self):
             created_by_user=self.request.couch_user.username,
         )
 
-    @staticmethod
-    def _convert_timestamp_to_utc(timestamp, timezone):
-        return UserTime(
-            timestamp,
-            tzinfo=pytz.timezone(timezone)
-        ).server_time().done()
+
+class EditDomainAlertView(BaseDomainAlertsView):
+    template_name = 'domain/admin/edit_alert.html'
+    urlname = 'domain_edit_alert'
+    page_title = gettext_lazy("Edit Project Alert")
+
+    @property
+    @memoized
+    def page_url(self):
+        return reverse(ManageDomainAlertsView.urlname, args=[self.domain])
+
+    @property
+    def page_context(self):
+        return {
+            'form': self.form
+        }
+
+    @cached_property
+    def form(self):
+        if self.request.method == 'POST':
+            return DomainAlertForm(self.request, self.request.POST)
+
+        alert = self._get_alert()
+        assert alert, "Alert not found"
```

Comment:
If this assertion fails, what will the user see?

Reply:
a 500 page. This is more of an edge case that we are trying to avoid. Hence added an assertion instead of having it fail later.

Reply:
I guess it's fine